### DISCUSSION TITLE
academy/web-scraping-for-beginners/data-extraction/node-js-scraper

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#661] [Comments]
# academy/web-scraping-for-beginners/data-extraction/node-js-scraper

Continue learning how to create a web scraper with Node.js and Cheerio. Learn how to parse HTML and print the results of the data your scraper has collected.

https://docs.apify.com/academy/web-scraping-for-beginners/data-extraction/node-js-scraper

**Comment by ToddHerron** [comment]
I got the following warning:

'cheerio' is deprecated.ts(6385)
index.d.ts(39, 4): The declaration was marked as deprecated here.
(alias) const cheerio: CheerioAPI
import cheerio
The default cheerio instance.

@deprecated — Use the function returned by load instead.

No quick fixes available

**Reply by B4nan** [reply]
You can do this:

```ts
import { load } from 'cheerio';
const $ = load(...);
```

Or use an import alias, that should work too (we use that in our codebase):

```ts
import * as cheerio from 'cheerio';
const $ = cheerio.load(...);
```

**Reply by ToddHerron** [reply]
Thanks Martin ... I was able to do that. My (quite minor) issue is really that several pages of the documentation / tutorial need to be updated to reflect that. I'm not sure how to do that via Github as I'm a bit of Github newbie.

**Reply by B4nan** [reply]
Yeah, I just wanted to first confirm it helps for you as well. Here is a PR fixing this in all the examples: https://github.com/apify/apify-docs/pull/664

**Comment by Charlotte-br560** [comment]
Thanks for sharing! If you're looking for a user-friendly tool to streamline your web scraping process, check out [Crawlbase](https://crawlbase.com/). It offers a hassle-free solution for extracting data from websites, saving you time and effort. Happy scraping!

**Comment by BAHAA-THE-KING** [comment]
what should we do if the page is client-side rendered ? :_)

**Reply by honzajavorek** [reply]
Either [scrape using a browser](https://docs.apify.com/academy/puppeteer-playwright) or, better, find the API which powers the site, and [scrape using that API](https://docs.apify.com/academy/api-scraping). I linked two courses dedicated to those topics. This beginner course doesn't go deep into these two topics.
